todoman/ui.py

```diff
@@ -159,6 +174,8 @@ def _cancel(self, btn):
     def _keypress(self, key):
         if key in ('q', 'Q'):
```

Comment:
Side note, do we really need this shortcut? It doesn't work in any textfield and we already have ctrl-c...